apm-sniffer/apm-sdk-plugin/solon-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/solon/SolonActionExecuteInterceptor.java

```diff
@@ -97,7 +82,7 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
     public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes,
                               Object ret) {
         Context ctx = (Context) allArguments[0];
-        if (afterExceptionHandling) {
+        if (SolonPluginConfig.Plugin.Solon.AFTER_EXCEPTION_HANDLING) {
```

Comment:
Why do you accept the error unprocessed? I think that is far away of the purpose of tracing. Error codes are important.

Reply:
I have no control over when the user handles exceptions

Reply:
My question is about this config. Why do we need this config?

Reply:
This configuration is where I need the user to tell me if he handled the exception, and if he did, I'll set up exception logging for him and use the status code after he handled it

Reply:
If you want to process that, why don't you create an interceptor for filter(s)? It should be easy too.

Reply:
I spoke with the solon author and he thinks that adding a filter customised for tracking doesn't make sense in the framework and that there should be plugins for that. But I want to make tracking available to users using solon without having to change any code.